src/components/fields/category-select-field.vue

```diff
@@ -89,6 +96,7 @@
 import {
   defineComponent, ref, Ref, computed, ComputedRef, PropType,
 } from 'vue';
+import InputField from '@/components/fields/input-field.vue';
 import { CategoryModel } from 'shared-types';
```

Comment:
Rewrite this component using `setup` syntax. There are already examples in the app so you can take a look at them